### PR TITLE
Package pringo.1.0

### PR DESCRIPTION
**Pull request by xavierleroy**
### `pringo.1.0`
Pseudo-random, splittable number generators
Pseudo-random number generators that support splitting and two interfaces: one stateful, one purely functional

---
* Homepage: https://github.com/xavierleroy/pringo
* Source repo: git+https://https://github.com/xavierleroy/pringo
* Bug tracker: https://github.com/xavierleroy/pringo/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

File: packages/pringo/pringo.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+description:
+  "Pseudo-random number generators that support splitting and two interfaces: one stateful, one purely functional"
+maintainer: "Xavier Leroy <xavier.leroy@college-de-france.fr>"
+authors: "Xavier Leroy <xavier.leroy@college-de-france.fr>"
+license: "GPL v2 with static linking exception"
+homepage: "https://github.com/xavierleroy/pringo"
+bug-reports: "https://github.com/xavierleroy/pringo/issues"
+depends: [
+  "ocaml" {>= "4.05.0"}
+  "ocamlfind"
```
Comment (mseri) [inline packages/pringo/pringo.1.0/opam:12]: Is `ocamlfind` just a build dependency?

File: packages/pringo/pringo.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+]
+build: make
+install: [make "install"]
+remove: [make "uninstall"]
```
Comment (mseri) [inline packages/pringo/pringo.1.0/opam:16]: This can be removed with opam2

File: packages/pringo/pringo.1.0/opam (hunk 1 above)
Comment (xavierleroy) [reply packages/pringo/pringo.1.0/opam:12]: This package is a library, and you need ocamlfind to link with it, so I figured it's not just a build dependency.

File: packages/pringo/pringo.1.0/opam (hunk 2 above)
Comment (xavierleroy) [reply packages/pringo/pringo.1.0/opam:16]: It wasn't clear to me how OPAM2's automatic uninstall deals with ocamlfind-installed libraries.

Comment (mseri) [reply packages/pringo/pringo.1.0/opam:16]: Opam diffs the file tree in the `.opam` folder before and after the installation, and notes down the changes. At uninstall stage it reverts those changes: the `remove` stanza is still useful if you are making changes to the content of pre-existing files that need to be reverted during the uninstall phase.

Comment (xavierleroy) [reply packages/pringo/pringo.1.0/opam]: Thanks for the explanations.  I removed the "remove" line (but managed to botch the log entry).

**Comment by mseri** [comment]
Thanks!

**Pull request merged**